parent/pom.xml

```
@@ -36,6 +36,7 @@
     <version.jackson>2.15.2</version.jackson>
     <version.xml.bind-api>2.3.3</version.xml.bind-api>
     <version.xml.jaxb-impl>2.3.6</version.xml.jaxb-impl>
+    <version.xml.jaxb-impl4>4.0.4</version.xml.jaxb-impl4>
```

🔧 `com.sun.xml.bind:jaxb-impl:4.0.5` is also [available](https://mvnrepository.com/artifact/com.sun.xml.bind/jaxb-impl) in case it's not causing any conflicts

```suggestion
    <version.xml.jaxb-impl4>4.0.5</version.xml.jaxb-impl4>
```